webgl/transcoder/build/basis_loader.js

```diff
@@ -1,494 +1,494 @@
-/*

```

Comment:
The changes to the *.MD files and this javascript, were these don to please `./format.sh`?

Reply:
All I did was run `./format.sh` and commit the results. So yeah.

Reply:
If I recall correctly, this is fixing the line endings of these files.